app/tweep_dm.py

```diff
@@ -0,0 +1,223 @@
+import json
+import os
+import re
+from typing import Dict, List
+
+import requests
+import tweepy
+from dotenv import load_dotenv
+from requests_oauthlib import OAuth1Session
+
+from app.scrap_paper import confirmation_reply, quick_reply_use_of_force
+
+load_dotenv()
+
+auth = tweepy.OAuthHandler(os.getenv("CONSUMER_KEY"),
+                           os.getenv("CONSUMER_SECRET"))
+auth.set_access_token(os.getenv("ACCESS_KEY"), os.getenv("ACCESS_SECRET"))
+api = tweepy.API(auth, wait_on_rate_limit=True)
+
+manual_twitter_auth = OAuth1Session(os.getenv("CONSUMER_KEY"),
+                                    os.getenv("CONSUMER_SECRET"),
+                                    os.getenv("ACCESS_KEY"),
+                                    os.getenv("ACCESS_SECRET"))
+
+# Just my own for now
+blue_witness_recipient_id = 271825982
+
```

Comment:
Remember to delete this before pushing.  This is more of a reminder than a comment.